ui/apps/platform/src/Containers/NetworkGraph/deployment/DeploymentFlows.tsx

```diff
@@ -0,0 +1,252 @@
+                                <Th width={30}>{columnNames.portAndProtocol}</Th>
+                            </Tr>
+                        </Thead>
+                        {flows.map((row, rowIndex) => {
+                            const isExpanded = isRowExpanded(row);
+                            return (
+                                <Tbody key={row.entity} isExpanded={isExpanded}>
+                                    <Tr>
+                                        <Td
+                                            expand={
+                                                row.children.length
+                                                    ? {
+                                                          rowIndex,
+                                                          isExpanded,
+                                                          onToggle: () =>
+                                                              setRowExpanded(row, !isExpanded),
+                                                          expandId: 'flow-expandable',
+                                                      }
+                                                    : undefined
+                                            }
+                                        />
+                                        <Td dataLabel={columnNames.entity}>
+                                            <Flex direction={{ default: 'row' }}>
+                                                <FlexItem>
+                                                    <div>{row.entity}</div>
+                                                    {row.type === 'Deployment' && (
+                                                        <div>
+                                                            <TextContent>
+                                                                <Text
+                                                                    component={TextVariants.small}
+                                                                >
+                                                                    in {`"${row.namespace}"`}
+                                                                </Text>
+                                                            </TextContent>
+                                                        </div>
+                                                    )}
+                                                    {row.type === 'External' && (
+                                                        <div>
+                                                            <TextContent>
+                                                                <Text
+                                                                    component={TextVariants.small}
+                                                                >
+                                                                    {`${row.children.length} active flows`}
```

Comment:
maybe you can do the conditional texts here instead of around the `<div><TextContent><Text component={TextVariants.small}>`